### PR TITLE
Folder Name Length Validation

### LINKED ISSUE #4420: Unlimited Folder Name Possible
### Environment
ToolJet Cloud

### What is the expected behaviour?
Folder name creation gives error when attempt to make a folder with over 25 chars.

### What is the current behaviour?
No backend validation, can create long folder names.

### How to reproduce the issue?

1. Click "Create Folder"
2. Open inspect tools (F12)
3. Delete the max characters = 25
4. Unlimited characters possible due to no backend validation.

### Screenshots or Screencast
image

### Any possible solutions?
Add backend validation.

### Can you identify the location in the driver source code where the problem exists?
Haven't looked.

### If the bug is confirmed, would you be willing to submit a PR?
Yes

### PR DESCRIPTION
**Pull request by Frenchtoast0**
Added frontend and backend validation.

fixes #4420

### CODE REVIEW

**Comment by Frenchtoast0** [comment]
https://github.com/ToolJet/ToolJet/issues/4420

**Comment by gsmithun4** [comment]
@Frenchtoast0 please fix the lint errors

File: server/src/services/folders.service.ts [hunk 1]
```diff
@@ -19,9 +19,15 @@ export class FoldersService {
     @InjectRepository(App)
     private appsRepository: Repository<App>,
     private usersService: UsersService
-  ) {}
+  ) { }
 
   async create(user: User, folderName): Promise<Folder> {
+    if (folderName.length < 1) {
+      throw new BadRequestException('Folder name cannot be shorter than 1 character');
```
Comment (gsmithun4) [inline server/src/services/folders.service.ts:26]: Shall we create a DTO([example](https://github.com/ToolJet/ToolJet/blob/develop/server/src/dto/version-create.dto.ts)) and add `@maxLength` validation?

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

File: server/src/services/folders.service.ts (hunk 1 above)
Comment (Frenchtoast0) [reply server/src/services/folders.service.ts:26]: Could that be in a new issue? I'm running out of time for Hacktoberfest.

**Comment by Frenchtoast0** [comment]
Lint errors should be fixed.

File: server/src/services/folders.service.ts [hunk 2]
```diff
@@ -22,6 +22,11 @@ export class FoldersService {
   ) {}
 
   async create(user: User, folderName): Promise<Folder> {
+    if (folderName.length < 1) {
+      throw new BadRequestException('Folder name cannot be shorter than 1 character');
```
Comment (gsmithun4) [inline server/src/services/folders.service.ts:26]:
```suggestion
    if (!folderName) {
      throw new BadRequestException('Folder name cannot be empty');
```

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

**Comment by Frenchtoast0** [comment]
I’m not sure why the lint check keeps getting stuck.

**Comment by gsmithun4** [comment]
@Frenchtoast0 Please rebase https://github.com/French-Toast-Coding/ToolJet/pull/1

**Review by gsmithun4 — APPROVED** [review]

**Comment by withshubh** [comment]
Hey @Frenchtoast0 
Thanks for contributing to ToolJet. :raised_hands: 
Please submit the form to claim your Hacktoberfest swag: https://airtable.com/shr1qR6xhqsWO7LD7

**Pull request merged**